Assets/Scripts/Character/CharacterMovement.cs

```diff
@@ -83,6 +90,11 @@ public class CharacterMovement : MonoBehaviour {
 
         // For debugging purposes
         velocity = _rigidbody.velocity;
+        velocityLocal = transform.InverseTransformDirection(velocity);
+        isMovingForward = velocityLocal.z > MAX_VELOCITY * 0.5;
```

Comment:
Same as my nitpick in PlayerInput, I think maybe it would be better to use a variable name instead of 0.5, even a variable local to the function. Not a big deal though.